react/src/lib/Alert/examples/Default.js

```diff
@@ -1,36 +1,41 @@
 import React from 'react';
 import {
+  Alert,
   AlertContainer,
   Button
 } from '@momentum-ui/react';
 export default class AlertDefault extends React.PureComponent {
   state = {
-    alertMessage: 'Who\'s awesome?  You are!'
+    alertMessage: 'Who\'s awesome?  You are!',
+    show: false
   }
   render() {
-    let alertContainer;
     return (
       <section>
         <div>
           <div className='row'>
             <Button
               ariaLabel='Click to Open'
-              onClick={() => alertContainer.info(
-                'Alert',
-                this.state.alertMessage,
-                () => alert('onHide info'),
-                { ariaLabel: 'Close Alert' }
-              )}
+              onClick={() => this.setState({ show: true })}
```

Comment:
would a more realistic example be adding the alert to an array state onClick, and then pop it off the array on dismiss?

Reply:
You could still keep track of an array in your app, and just leave all alerts with the show prop as true by default.  This is specifically for the example.